README.md

```diff
@@ -123,8 +123,11 @@ Note that these builds may be unstable since the main branch is a work in progre
 
 To get nightly downloads, please
 [click here](https://github.com/logisim-evolution/logisim-evolution/actions/workflows/nightly.yml)
-and browse to the last successful run of `Nightly` worker, which should be on top. Note that due to Github internals,
-all files are provided as ZIP archives. You must unzip the downloaded file to get the installation package.
+and browse to the last successful run of the `Nightly` worker.
+It should be the first from the top that took a several minutes to run.
```

Comment:
"[...] took several minutes to run."?

Reply:
I reworded this section to, hopefully, make it clearer.